src/app/page.jsx

```diff
@@ -0,0 +1,22 @@
+'use client'
+
+import Message from '../components/message'
+
+export default function Home() {
+  return (
+    <main className="flex flex-col gap-3 p-3">
+      <Message timeCreated="16.27">
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quae
+        architecto quis iure ex id voluptatum reprehenderit quod cum laboriosam.
+        Dicta temporibus quis sed assumenda tempore perferendis voluptas magnam
+        minima dignissimos! Dignissimos porro ea incidunt quae blanditiis,
+        exercitationem deserunt beatae omnis temporibus officiis, consequatur
+        nulla accusamus commodi officia minus, ullam unde optio nisi similique
+        ab maiores? Perferendis aliquid minus laudantium error.
+      </Message>
+      <Message timeCreated="16.27">Hello, i'm a message! 😎</Message>
```

Comment:
Suggest renaming `timeCreated` to `createdAt` and make it a number. i.e. [Unix timestamp](https://en.wikipedia.org/wiki/Unix_time)

Reply:
Ok

Reply:
Yes, you can [Intl API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl).

```js 
const formatter = new Intl.DateTimeFormat('en', {
  hour: '2-digit',
  minute: '2-digit',
  hour12: false,
});

console.log(formatter.format(Date.now()));

```